src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
+        <input id="cellsynt-login" v-model="$parent.notification.cellsyntLogin" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-key" class="form-label">{{ $t("Password") }}</label>
+        <HiddenInput id="cellsynt-key" v-model="$parent.notification.cellsyntPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-Originatortype" class="form-label">{{ $t("Originator type") }}</label>
+        <select id="cellsynt-Originatortype" v-model="$parent.notification.cellsyntOriginatortype" :required="true" class="form-select">
+            <option value="alpha">{{ $t("Alpha (recommended)") }}</option>
+            <option value="numeric">{{ $t("Numeric") }}</option>
+        </select>
+        <div class="form-text">
+            <p><b>{{ $t("Alpha (recommended)") }}:</b><br /> {{ $t("Alphanumeric string (max 11 characters). The following characters are guaranteed to work: a-z, A-Z and 0-9. Other characters may work but functionality can not be guaranteed. Recipients can not reply to messages with alphanumeric originators.") }}</p>
+            <p><b>{{ $t("Numeric") }}:</b><br /> {{ $t("Numeric value (max 15 digits) with telephone number on international format without leading 00 (example UK number 07920 110 000 should be set as 447920110000). Receiving mobile phone will add a leading + sign and thus see the originator as a normal mobile phone number (+447920110000). Therefore it is also possible to reply to the message.") }}</p>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-originator" class="form-label">{{ $t("Originator") }}</label>
+        <input id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="text" class="form-control" required>
+        <div class="form-text"><p>{{ $t("Identifier which will be visible on recipient's mobile phone as originator of the message. Allowed values and function depends on parameter originatortype.") }}</p></div>
```

Comment:
Can be shortend a bit

```suggestion
        <div class="form-text"><p>{{ $t("Visible on recipient's mobile phone as originator of the message. Allowed values and function depends on parameter originatortype.") }}</p></div>
```